modules/core/src/main/scala/ch/epfl/scala/debugadapter/internal/evaluator/RuntimeEvaluationHelpers.scala

```diff
@@ -19,7 +19,7 @@ private[evaluator] object Helpers {
     val tpe = classLoader
       .mirrorOfLiteral(literal.value)
       .map(_.value.`type`)
-      .getResult
+      .extract
```

Comment:
There is a memory leak here. If the lit is a string it will get created in the heap, but will never get garbage collected.
```suggestion
      .getResult
```